spring-data-vmware-gemfire/src/main/java/org/springframework/data/gemfire/config/annotation/AutoRegionLookupConfiguration.java

```diff
@@ -117,7 +114,7 @@ public void registerBeanDefinitions(AnnotationMetadata importingClassMetadata, B
 	@SuppressWarnings("unused")
 	private boolean isEnabled(String enabled) {
 
-		enabled = StringUtils.trimWhitespace(enabled);
+		enabled = enabled.strip();
```

Comment:
why this change?

Reply:
StringUtils.trimWhitespace was deprecated in favor of String.strip, so I figured I'd clean it up while I was in the area.